src/format/command.rs

```diff
@@ -103,7 +107,7 @@ fn format_file(path: PathBuf) -> Result<Formatted, Error> {
     })
 }
 
-pub fn format_stdin(_check: bool) -> Result<(), Error> {
+pub fn format_stdin(check: bool) -> Result<(), Error> {
```

Comment:
What does the `check` argument refer to? Is there a more descriptive name?

Reply:
It refers to the `--check` flag. I've requests that this argument be removed from this function and moved to the higher level.